conans/client/installer.py

```diff
@@ -312,6 +317,19 @@ def _node_concurrently_installed(self, node, package_folder):
             if node.update_manifest == read_manifest:
                 return True
 
+    def _handle_node_editable(self, node):
+        # TODO: I need a valid 'package_folder', but I'm not going to use it
+        package_folder = self._client_cache.conan(node.conan_ref)
```

Comment:
I think I'd map this one to something in the editable space. Maybe the root of the edited package.

Reply:
This comes from a previous implementation when I needed a _valid_ path like "...lib/version/user/channel", now I will use the edited package one.